core/ui/card_ui/disks/steam_removable_media.gd

```diff
@@ -0,0 +1,66 @@
+extends ScrollContainer
+
+var steam_disks := load("res://core/systems/disks/steam_removable_media_manager.tres") as SteamRemovableMediaManager
+
+const drive_card_scene: PackedScene = preload("res://core/ui/components/drive_card.tscn")
+
+
+var logger := Log.get_logger("SteamRemovableMedia", Log.LEVEL.INFO)
+
+@onready var container: HFlowContainer = $%DriveCardContainer
+@onready var focus_group: FocusGroup = $%FocusGroup
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
```

Comment:
We should only show this menu if certain prerequisites are met (i.e. if UDisks2 is available, etc.)